src/main/java/org/zalando/nakadi/Application.java

```diff
@@ -2,10 +2,13 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
 import org.springframework.context.annotation.ComponentScan;
+import org.zalando.nakadi.config.FeaturesConfig;
 
 @SpringBootApplication
 @ComponentScan(basePackages = {"org.zalando.nakadi", "org.zalando.nakadi.config"})
+@EnableConfigurationProperties(FeaturesConfig.class)
```

Comment:
Usually, it is in configuration, where you use it, in this case I would put on RepositoriesConfig and moved FeaturesConfig into RepositoriesConfig, it will be more localised to the class where it is used. What do you think?

Reply:
You are right, I moved it to RepositoriesConfig